builtin/mainmenu/dlg_config_world.lua

```diff
@@ -238,6 +245,20 @@ local function handle_buttons(this, fields)
 				list[i].enabled = true
 			end
 		end
+		enabled_all = true
+		return true
+	end
+	
+	if fields["btn_disable_all_mods"] then
+		local list = this.data.list:get_raw_list()
+
+		for i=1,#list,1 do
+			if list[i].typ ~= "game_mod" and
+				not list[i].is_modpack then
```

Comment:
I don't think this line needs to be broken in the first place it is under 80 characters.